operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -126,4 +130,14 @@ default boolean closeClientOnStop() {
   default ObjectMapper getObjectMapper() {
     return OBJECT_MAPPER;
   }
+
+  default <T extends DependentResource<?, ?>> T createFrom(DependentResourceSpec<T, ?> spec) {
```

Comment:
This feels a little weird, that a configuration service is responsible for instantiating a Dependent Resource.

Reply:
True, would be better to add a factory if we want to go that way. Putting it directly on `ConfigurationService` was a shortcut.